.github/workflows/main.yaml

```diff
@@ -0,0 +1,48 @@
+name: Main
+
+on:
+    push:
+        branches:
+            - main
+            - master
+    pull_request:
+        branches:
+            - main
+            - master
+
+jobs:
+    jest:
+        name: Jest
+        runs-on: ${{ matrix.os }}
+
+        strategy:
+            matrix:
+                os: [ubuntu-latest]
+                node: [14]
```

Comment:
This matrix is useless as it only has a single value. In this case I'd say it's easier to hardcode the values or add more entries (like testing both node latest and LTS). (the OS should always stay the same though, since the project doesn't use platform specific features)